internal/eva/create_organizaion_unit.go

```diff
@@ -0,0 +1,63 @@
+package eva
```

Comment:
Rather than having a file per verb (create) would it not be simpler to keep it all grouped by resource?
`internal/eva/organization_unit(s).go` rather than `internal/eva/create_organization_unit.go`

Reply:
this is a fair observation 🤔 
I was going for each EVA `endpoint` gets their own file. But this could indeed turn massive over time. 

I think its a good suggestion. Will do 👍🏼